homeassistant/components/sensor/fail2ban.py

```diff
@@ -26,7 +23,6 @@
 
 DEFAULT_NAME = 'fail2ban'
 DEFAULT_LOG = '/var/log/fail2ban.log'
-SCAN_INTERVAL = timedelta(seconds=120)
```

Comment:
Don't remove this. We still want the default update interval set to 2 minutes, right?

Reply:
Restored that in latest commit